test/examples/TrafficLight.test.js

```diff
@@ -0,0 +1,24 @@
+var assert = require('assert');
```

Comment:
Let's move tests for examples to examples directory. For instance, you have
`/examples/states/trafficLight`
`/examples/states/trafficLight/_tests`
`/examples/states/trafficLight/_tests/TrafficLighe.test.js`

Also it's better to add `unit` suffix to the file. Because there also will be functional and integration tests.